### PR TITLE
Feature/exec time

### PR DESCRIPTION
**Pull request by marcaurele**
Add in scenario execution a "started" timestamp and for each client executions. Those values are display in the scenario's replays view.

### CODE REVIEW

File: src/warp/client/views.cljs [hunk 1]
```diff
@@ -120,21 +120,27 @@
...
                       ^{:key (str "step-" index)}
                       (success-output success))
                     [(success-output (every? :success steps))])))
 
 (defn replay-summary
-  [{:keys [clients accepted refused total scenario id]}]
+  [{:keys [clients accepted refused total scenario id started]}]
   (panel
-   [:span "Run:" (code id)]
+   [:span "Run:" (code id) " @ " started]
```
Comment (marcaurele) [inline src/warp/client/views.cljs]: I cannot use `(sanitize-date started)` here. why?

Comment (marcaurele) [inline src/warp/client/views.cljs]: ack

Comment (pyr) [inline src/warp/client/views.cljs]: you can test with my update to `project.clj`

**Comment by pyr** [comment]
LGTM apart from the nitpick.

**Comment by marcaurele** [comment]
Which nitpick?

**Pull request closed without merging**